src/axom/sidre/interface/c_fortran/utilSidre.cpp

```diff
@@ -0,0 +1,297 @@
+// utilSidre.cpp
+// This file is generated by Shroud 0.14.0. Do not edit.
+//
+// Copyright (c) 2017-2025, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <string>
+#include "axom/sidre/core/DataStore.hpp"
+#include "typesSidre.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Release library allocated memory.
+void SIDRE_SHROUD_memory_destructor(SIDRE_SHROUD_capsule_data *cap)
+{
+  void *ptr = cap->addr;
+  switch(cap->idtor)
```

Comment:
Where are these `idtor` values mapped from integers to types?
e.g. 
* 1 -> `string`
* 2->`DataStore`